tests/ops/conftest.py

```diff
@@ -250,3 +251,13 @@ def subject_identity_verification_not_required():
     config.execution.subject_identity_verification_required = False
     yield
     config.execution.subject_identity_verification_required = original_value
+
+
+@pytest.fixture(scope="session", autouse=True)
+def event_loop():
```

Comment:
could you explain why this is needed specifically for tests?

Reply:
It happens to be the same issue we are solving with the async log. When running the async tests it will sometimes start up multiple event loops and cause errors. I have found that using this fixture in my other async code bases has prevent the error.